docs/logs/getting-started-aspnetcore/README.md

```diff
@@ -0,0 +1,150 @@
+LogRecord.CategoryName:            Microsoft.Hosting.Lifetime
+LogRecord.Severity:                Info
+LogRecord.SeverityText:            Information
+LogRecord.Body:                    Now listening on: {address}
+LogRecord.Attributes (Key:Value):
+    address: http://localhost:5000
+    OriginalFormat (a.k.a Body): Now listening on: {address}
+LogRecord.EventId:                 14
+LogRecord.EventName:               ListeningOnAddress
+
+Resource associated with LogRecord:
+service.name: getting-started-aspnetcore
+service.instance.id: d62727d2-3e80-4780-a984-e85fc861e321
+telemetry.sdk.name: opentelemetry
+telemetry.sdk.language: dotnet
+telemetry.sdk.version: 1.6.0-rc.1.24
+
+LogRecord.Timestamp:               2023-09-06T23:00:46.1639248Z
+LogRecord.TraceId:                 3507087d60ae4b1d2f10e68f4e40784a
+LogRecord.SpanId:                  c51be9f19c598b69
+LogRecord.TraceFlags:              None
+LogRecord.CategoryName:            Program
+LogRecord.Severity:                Info
+LogRecord.SeverityText:            Information
+LogRecord.Body:                    Food `{name}` price changed to `{price}`.
+LogRecord.Attributes (Key:Value):
+    name: artichoke
+    price: 9.99
+    OriginalFormat (a.k.a Body): Food `{name}` price changed to `{price}`.
+LogRecord.EventId:                 2
+LogRecord.EventName:               FoodPriceChanged
+LogRecord.ScopeValues (Key:Value):
+[Scope.0]:SpanId: c51be9f19c598b69
+[Scope.0]:TraceId: 3507087d60ae4b1d2f10e68f4e40784a
+[Scope.0]:ParentId: 0000000000000000
+[Scope.1]:ConnectionId: 0HMTF5KCP0BU9
+[Scope.2]:RequestId: 0HMTF5KCP0BU9:00000003
+[Scope.2]:RequestPath: /
+
+Resource associated with LogRecord:
+service.name: getting-started-aspnetcore
+service.instance.id: d62727d2-3e80-4780-a984-e85fc861e321
+telemetry.sdk.name: opentelemetry
+telemetry.sdk.language: dotnet
+telemetry.sdk.version: 1.6.0-rc.1.24
+```
+
+Congratulations! You are now collecting logs using OpenTelemetry.
+
+What does the above program do?
+
+The program has added OpenTelemetry as a [logging
+provider](https://learn.microsoft.com/dotnet/core/extensions/logging-providers)
+to the existing logging pipeline. OpenTelemetry SDK is then configured with a
+`ConsoleExporter` to export the logs to the console. In addition,
+`OpenTelemetryLoggerOptions.IncludeScopes` is enabled so the logs will include
+the [log
+scopes](https://learn.microsoft.com/dotnet/core/extensions/logging#log-scopes).
+From the console output we can see the log scopes that are coming from the
+ASP.NET Core framework, and we can see logs from both our logger and the ASP.NET
+Core framework loggers.
+
+The example has demonstrated the best practice from ASP.NET Core by injecting
+generic `ILogger<T>`:
+
+```csharp
+app.MapGet("/", (ILogger<Program> logger) =>
+{
+    logger.FoodPriceChanged("artichoke", 9.99);
+
+    return "Hello from OpenTelemetry Logs!";
+});
+```
+
+To achieve structured logging and high performance, [compile-time logging source
```

Comment:
You still get structured logging via other APIs as well. What you don't get in the other APIs are type checked parameters.

Reply:
I've reworded this section, PTAL.